src/twisted/mail/newsfragments/9180.bugfix

```diff
@@ -0,0 +1 @@
+Sending a list of recipients with twisted.smtp.SenderFactory has been fixed.  This fixes a problem found when running buildbot.
```

Comment:
Where is tardyp's review?  The  convention is for the review to be left on the PR.  Failing that, the trac ticket.  I don't see it in either place.  Am I missing it somewhere?